client/components/ZoopsIndex.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const ZoopsIndex = () => {
```

Comment:
nit: I'm not so sure about this naming?  Seems a little non-standard. I would expect something like ZoopsList, ZoopsView, ZoopsPage - or even just MainView or MainPage

Reply:
Josh mentioned that this was a naming convention he taught at App Academy. Although it seemed like he was unsure if "index" was the exact word when we were working through it. @jvaneyken Was "index" the word they suggested at App Academy for this sort of page?